x/concentrated-liquidity/swaps.go

```diff
@@ -394,7 +394,8 @@ func (k Keeper) calcOutAmtGivenIn(ctx sdk.Context,
 
 	// coin amounts require int values
 	// round amountIn up to avoid under charging
-	amt0 := tokenAmountInSpecified.Sub(swapState.amountSpecifiedRemaining).RoundInt()
+	// round amountOut down to avoid over refunding.
```

Comment:
nit: move this comment to right above `amt1` – did a double take because I thought it was referring to `amt0` (which rounds up)

Reply:
done